src/ts/components/subtitleoverlay.ts

```diff
@@ -416,3 +418,82 @@ class ActiveSubtitleManager {
     this.activeSubtitleCueCount = 0;
   }
 }
+
+class SubtitleRegionContainerManager {
+  private subtitleRegionContainers: { [region: string]: SubtitleRegionContainer } = {};
+  private subtitleOverlay: SubtitleOverlay;
+
+  /**
+   * @param subtitleOverlay Reference to the subtitle overlay for adding and removing the containers.
+   */
+  constructor(subtitleOverlay: SubtitleOverlay) {
```

Comment:
Change the signature to `private subtitleOverlay` to get rid of the explicit private field definition